app/services/LocationService.js

```diff
@@ -212,10 +213,15 @@ export default class LocationServices {
     const locationData = new LocationData();
     // handles edge cases around Android where start might get called again even though
     // the service is already created.  Make sure the listeners are still bound and exit
-    if (isBackgroundGeolocationConfigured) {
-      BackgroundGeolocation.start();
-      return;
-    }
+    let result = BackgroundGeolocation.checkStatus(({ isRunning }) => {
```

Comment:
This piece of logic (L216-L224) feels like it's getting complicated enough to pull into its own method.

I think it would be a bit cleaner to just do the boolean status check in the method and start BackgroundGeolocation afterwards. E.g.


```js
if (this.isBackgroundGeolocationRunning()) {
  return 
} else {
  BackgroundGeolocation.start()
}
```

Reply:
Also - I believe we can get rid of [isBackgroundGeolocationConfigured](https://github.com/tripleblindmarket/covid-safe-paths/pull/725/files#diff-3a190ea6886abecabc4984cc71181843R12) and all references to it in the file now?

Reply:
will do